gopowerstore_types.go

```diff
@@ -106,6 +106,11 @@ func (err *APIError) VolumeAttachedToHost() bool {
 	return err.StatusCode == http.StatusUnprocessableEntity
 }
 
+// VolumeDetachedFromHost returns true if API error indicate that volume is detached from host
+func (err *APIError) VolumeDetachedFromHost() bool {
```

Comment:
VolumeAttachedToHost() also does the same thing.. Is there any specific reason to have a separate method for Detach or could that be renamed to perform both the operations? May not be in the scope of this defect

Reply:
Yes, Niranjan this `VolumeAttachedToHost()`  also does the same thing. To check the volume detached from the host we cannot use the volumeattachedfromhost function, It won't be good for the readability, we have intentionally created that function for the detach functionality, (As we are seeing some inconsistent behavior from the array side)
I agree that there is little code cleanup needed here in that file. we will pick that in the future.

Thank you